OpenRA.Platforms.Default/ThreadedGraphicsContext.cs

```diff
@@ -526,6 +531,12 @@ public void SetData(Vertex[] vertices, int length)
 			device.Post(setData1, (buffer, length));
 		}
 
+		public void SetData(ref Vertex[] vertices, int length)
+		{
+			device.Post(setData1, (vertices, length));
```

Comment:
Can we please have a comment here explaining that we are switching out `vertices` so that we can hand it to the worker thread without worrying about race conditions?

Reply:
Looks like this was silently updated. Does the new comment suffice in your eyes?
( `// PERF: We pass vertices without copying to the render thread and return a new array to the caller.`)